app/src/main/java/net/bible/android/control/backup/BackupControl.kt

```diff
@@ -113,7 +93,7 @@ class BackupControl @Inject constructor() {
 
         if (ok) {
             Log.d(TAG, "Copied database to chosen backup location successfully")
-            Dialogs.instance.showMsg(R.string.backup_success, uri.path)
```

Comment:
Why is this removed? I would think it is useful to see where the file was saved.

Reply:
Check out screenshots on #756.

Reply:
I missed that issue, good change then